src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -361,8 +367,8 @@ protected virtual void PostShutdownCore(bool shutdownHost)
         public struct StartConfiguration
         {
             public string Context { get; set; }
-            public string DynamoCorePath { get; set; }
             public IPreferences Preferences { get; set; }
+            public IPathResolver PathResolver { get; set; }
```

Comment:
`DynamoModel.StartConfiguration.DynamoCorePath` is removed, and replaced by `IPathResolver` implementation from clients. Internally, `DynamoModel` will always retain this path as the containing folder of `DynamoCore.dll` for all purposes of path resolution.